mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -2710,13 +2776,25 @@ fn electrum_connect(
 }
 
 fn electrum_request(
+    client: &ElectrumClient,
```

Comment:
Do you think if we `JsonRpcErrorType` from this function instead of `JsonRpcError` will be better? We will not need this function parameter then. Plus, it will reduce the code significantly.

Reply:
yes please!